text/0000-sigstore-integration.md

```diff
@@ -0,0 +1,387 @@
+
+The public Sigstore service has out of the box integration with many of the above-mentioned IdP’s and so use of the public service is quite straightforward/simple for any individual developer. It is also worth noting that businesses that choose to implement an internal Sigstore signing service can very easily wire up the Fulcio component of Sigstore to the internal IdP for private signing / verification operations.
+
+Implementation of Sigstore signing does need to consider (but will not directly address) regulatory requirements related to PII. For example, the EU GDPR requires that personally identifiable information (PII) must be removable (email addresses fall into this classification). However, the Rekor component of Sigstore is implemented on top of a Transparency Log which is by definition and design immutable. Effectively this means that for now, people with privacy concerns should not be using actual personal identities tied to their GH account and email. 
+
+It is very important to acknowledge that while this situation with personal identity persists, crate artifacts produced on the desktop must always be considered as unsafe, even if signed. In the same way that one can use “burner’ GH and email accounts to protect their privacy, malicious actors can as well to introduce attacking software. There are other operational imperatives related to the safe consumption of any OSS, but that is beyond the scope of this document. 
+
+However, in practice most OSS projects where there are more than a handful of contributors will very likely use CI/CD (Build) capabilities provided through an aaS model. In this scenario, identity in the context of the build activity now shifts to the NPE model. Here, providers capable of authenticating to the public Sigstore service via organic OIDC capabilities. Mainstream commercial build plants running aaS are de-facto subject to various standards and certifications (both operational and security), and so can provide certain levels of attestation metadata related to the production of the artifact. Further, they of course have no privacy concerns. It is for these reasons, and others that the team at GH implementing Sigstore tooling for the NPM ecosystem have chosen to focus on the aaS build scenarios as they talk with and evangelize that focus as a broader pattern. This is a pattern that this RFC agrees with and adopts.
+
+## Authentication
+
+Sigstore uses the OIDC pattern and ecosystem as the basis for authentication against the signing issuer (Fulcio). We assume that most are familiar with the OIDC pattern and tooling and so will not address those topics here, with the exception of the OIDC identity payload attributes. The certificates used for signing of course need to have an associated identity:
+
+* In the case of an individual developer signing from his desktop, Sigstore will embed the email address of the signing individual in the metadata delivered to the Rekor service. 
```

Comment:
I can only hope that your first language is not English, because otherwise, using "his" to describe a hypothetical developer is an extremely bad look. The proper term is their.